layers/meta-balena-beaglebone/conf/layer.conf

```diff
@@ -26,9 +26,23 @@ PREFERRED_PROVIDER_u-boot:beagleboard-xm = "u-boot"
 KERNEL_IMAGETYPES:beaglebone-ai64 = "Image"
 KERNEL_CLASSES:beaglebone-ai64 = "kernel-uimage"
 
+PREFERRED_PROVIDER_virtual/kernel:beagleplay = "linux-bb.org"
+PREFERRED_PROVIDER_virtual/bootloader:beagleplay = "u-boot-ti-staging"
+PREFERRED_PROVIDER_u-boot:beagleplay = "u-boot-ti-staging"
```

Comment:
This can be removed

Reply:
The `u-boot-ti-staging` provider, along with the preferred versions, seem to be needed (at least until the `Beagleplay` support is added to the `git.ti.com` mainline kernel).